vpdq/python/vpdq.pyx

```diff
@@ -3,12 +3,15 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 import cv2
 import typing as t
+import subprocess
 
 from dataclasses import dataclass
 from libcpp cimport bool
 from libcpp.vector cimport vector
 from libcpp.string cimport string
 
+subprocess.check_call("ffmpeg")
```

Comment:
blocking: This could use some more comments explaining what this call means. It otherwise might be mistaken for a undeleted test statement. 

Additionally, this might be a surprising error to a caller. You may want to convert it into another error type with a more specific error message. You could provide installation instructions in the error, for example. 

FileNotFound seems like a reasonable exception to use for this case (ffmpeg's file is literally not found). You may want to catch and throw your own version with more instructions, however.